### PR TITLE
WT-7070 Triage column store HS test failures

### PR DESCRIPTION
**Pull request by Siddhartha8899**

### CODE REVIEW

File: test/suite/test_hs04.py [hunk 1]
```diff
@@ -71,6 +71,11 @@ def get_stat(self, stat):
         stat_cursor.close()
         return val
 
+    def create_key(self, i):
```
Comment (sulabhM) [inline test/suite/test_hs04.py:74]: why add this function if not used anywhere?

File: test/suite/test_hs09.py [hunk 2]
```diff
@@ -117,6 +117,8 @@ def test_uncommitted_updates_not_written_to_hs(self):
 
         self.check_ckpt_hs(value2, value1, 2, 3)
 
+    # Remove for column store until prepared updates is implemented for column store
+    # WT-6061
```
Comment (sulabhM) [inline test/suite/test_hs09.py:121]: There is already a comment about removed till WT-6061

File: test/suite/test_hs11.py [hunk 3]
```diff
@@ -73,41 +83,44 @@ def test_non_ts_updates_clears_hs(self):
...
+        # WT-5545
         # Ensure that we blew away history store content.
-        for ts in range(1, 5):
-            self.session.begin_transaction('read_timestamp=' + timestamp_str(ts))
-            for i in range(1, 10000):
-                if i % 2 == 0:
-                    if self.update_type == 'deletion':
-                        cursor.set_key(str(i))
-                        self.assertEqual(cursor.search(), wiredtiger.WT_NOTFOUND)
+        if self.key_format != 'r':
```
Comment (sulabhM) [inline test/suite/test_hs11.py:103]: Probably easier to return from the function if column store:

```
if self.key_format == 'r':
    return
```

**Review by sulabhM — CHANGES_REQUESTED** [review]

File: test/suite/test_hs02.py [hunk 4]
```diff
@@ -42,8 +42,10 @@ class test_hs02(wttest.WiredTigerTestCase):
     session_config = 'isolation=snapshot'
 
     key_format_values = [
-        ('column', dict(key_format='r')),
-        ('string', dict(key_format='S'))
+        ('string', dict(key_format='S')),
+        # Remove for column store until rollback to stable is implemented for column store
```
Comment (alisonfel) [inline test/suite/test_hs02.py:46]: Styling/Formatting - I think these sets of comments need to be a completed sentence (ended with a period '.') ?

File: test/suite/test_hs02.py [hunk 5]
```diff
@@ -40,6 +41,14 @@ class test_hs02(wttest.WiredTigerTestCase):
     conn_config = 'cache_size=50MB,log=(enabled)'
     session_config = 'isolation=snapshot'
 
+    key_format_values = [
+        ('string', dict(key_format='S')),
+        # Remove for column store until rollback to stable is implemented for column store
```
Comment (tammybailey) [inline test/suite/test_hs02.py:46]: Generally, we preface these types of comments with a `FIXME-WT-XXXX`. Perhaps consider rewording as follows:

`# FIXME-WT-XXXX: Uncomment the column store scenario when rollback to stable support is complete.`

Also, WT-5545 is not the correct ticket to reference here. I believe it should be WT-7120.

File: test/suite/test_hs11.py [hunk 6]
```diff
@@ -73,41 +83,44 @@ def test_non_ts_updates_clears_hs(self):
...
         self.session.checkpoint()
 
         # Now apply an update at timestamp 10.
         for i in range(1, 10000):
             self.session.begin_transaction()
-            cursor[str(i)] = value2
+            cursor[self.create_key(i)] = value2
             self.session.commit_transaction('commit_timestamp=' + timestamp_str(10))
 
+        # Remove for column store until rollback to stable is implemented for column store
```
Comment (tammybailey) [inline test/suite/test_hs11.py:100]: Please see previous comment.

**Review by tammybailey — CHANGES_REQUESTED** [review]

File: test/suite/test_hs02.py (hunk 5 above)
Comment (Sean04) [reply test/suite/test_hs02.py]: No worries, just changed all our other comments to match this format.

File: test/suite/test_hs21.py [hunk 7]
```diff
@@ -0,0 +1,165 @@
...
+
+from helper import copy_wiredtiger_home
+import wiredtiger, wttest
+from wtdataset import SimpleDataSet
+from wtscenario import make_scenarios
+
+def timestamp_str(t):
+    return '%x' % t
+
+# test_hs01.py
```
Comment (kommiharibabu) [inline test/suite/test_hs21.py:37]: Does this test look almost the same as test_hs01? Is it mistakenly added? If not update the test name and description for this test.

Comment (Sean04) [reply test/suite/test_hs21.py:37]: Yeah sorry that was mistakenly added.

**Review by alisonfel — APPROVED** [review]
LGTM 👍

File: test/suite/test_hs11.py [hunk 8]
```diff
@@ -73,32 +83,37 @@ def test_non_ts_updates_clears_hs(self):
...
         self.session.checkpoint()
 
         # Now apply an update at timestamp 10.
         for i in range(1, 10000):
             self.session.begin_transaction()
-            cursor[str(i)] = value2
+            cursor[self.create_key(i)] = value2
             self.session.commit_transaction('commit_timestamp=' + timestamp_str(10))
 
+        # FIXME-WT-7120 Remove for column store until rollback to stable is implemented for column
```
Comment (tammybailey) [inline test/suite/test_hs11.py:100]: Missing a colon after `FIXME-WT-7120` and a period at the end of the sentence.

File: test/suite/test_hs11.py [hunk 9]
```diff
@@ -73,32 +83,37 @@ def test_non_ts_updates_clears_hs(self):
...
+        # store
         # Ensure that we blew away history store content.
```
Comment (tammybailey) [inline test/suite/test_hs11.py:102]: Should this comment be moved to line 105?

**Review by sulabhM — APPROVED** [review]

File: test/suite/test_hs13.py [hunk 10]
```diff
@@ -38,7 +38,8 @@ class test_hs13(wttest.WiredTigerTestCase):
     conn_config = 'cache_size=2MB,statistics=(all),eviction=(threads_max=1)'
     session_config = 'isolation=snapshot'
     key_format_values = [
-        # The commented columnar tests needs to be enabled once columnar Modify type update is fixed in (WT-5550).
+        # FIXME-WT-5550 The commented columnar tests needs to be enabled once columnar Modify type
```
Comment (tammybailey) [inline test/suite/test_hs13.py:41]: Colon missing after `FIXME-WT-5550`; Modify -> modify.

File: test/suite/test_hs12.py [hunk 11]
```diff
@@ -38,8 +38,8 @@ class test_hs12(wttest.WiredTigerTestCase):
     conn_config = 'cache_size=2MB,statistics=(all),eviction=(threads_max=1)'
     session_config = 'isolation=snapshot'
     key_format_values = [
-        # The commented columnar tests needs to be enabled once columnar
-        # Modify type update is fixed in (WT-5550).
+        # FIXME-WT-7120:The commented columnar tests needs to be enabled once columnar
```
Comment (tammybailey) [inline test/suite/test_hs12.py:41]: Add space after colon; Modify -> modify.

**Review by tammybailey — APPROVED** [review]
LGTM pending a few small style fixes.

**Pull request merged**